src/projectBank2/Bank.java

```diff
@@ -0,0 +1,61 @@
+package projectBank2;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.Map;
+import java.util.TreeMap;
+
+public class Bank {

```

Comment:
I would rename it to BankService, so that it clearly indicates that this class is actually doing something as is not dto.